docs/file-name.js

```diff
@@ -0,0 +1,6 @@
+class JsClass {
+    constructor() {
+    }
+}
+
+export default JsClass
```

Comment:
@MykolaGolubyev it would also be nice to automatically add `\n` at the end of each generated file to satisfy github 🥇

Reply:
will add. To clarify, this file is a part of auto scaffolded znai example project. You probably should not use it/commit it.

Reply:
Thanks. I figured not all generated files will end up in the repo but i'd still appreciate if it by default made github happy 🥇